<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="save_button">
      <property name="text">
       <string>img</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="open_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>10</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>img</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="bold_button">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>35</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
      <widget class="QPushButton" name="italic_button">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>0</y>
         <width>35</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
      <widget class="QPushButton" name="underlined_button">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>35</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>U</string>
       </property>
      </widget>
      <widget class="QPushButton" name="strikeout_button">
       <property name="geometry">
        <rect>
         <x>105</x>
         <y>0</y>
         <width>35</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <strikeout>true</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>abc</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QTextEdit" name="main_text"/>
    </item>
    <item row="0" column="1">
     <widget class="QFontComboBox" name="font_combo_box"/>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="font_size_box"/>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save file</string>
     </property>
     <addaction name="actionSave_2"/>
     <addaction name="actionSave_as"/>
    </widget>
    <addaction name="actionOpen_file"/>
    <addaction name="menuSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
